packages/mc-html-template/README.md

```diff
@@ -40,6 +40,19 @@ At the moment we define the following placeholders:
 
 This method will attempt to load and parse the `env.json` file, performing some validation and returning the parsed JSON.
 
+The environment configuration JSON file, usually called `env.json`, can contain placeholders for environment variables. This can be useful to avoid duplication between various `env.json` files for multiple different environments. Placeholders are specified with an expansion like syntax `${}` while additionally prefixing the environment variable name with `env:`.
```

Comment:
Maybe we can just link to the live docs from here, so that we don't duplicate it.